rpcd/playbooks/roles/filebeat/defaults/main.yml

```diff
@@ -57,6 +57,24 @@ multiline_openstack_pattern: '^[0-9-]{10} +[0-9:\.]+ +[0-9]+ +[A-Z]+ +[A-Za-z0-9
 #
 #                                  Date        Time      Date         Time     PID     Level   Python Module   Request Id
 multiline_keystone_error_pattern: '[0-9-]{10} +[0-9:\.]+ [0-9-]{10} +[0-9:\.]+ [0-9]+ +[A-Z]+ +[A-Za-z0-9\._]+ \[|Traceback'
+# NOTE(d34dh0r53): There are multiple patterns of multiline entries
+# that appear in the various swift logs. These patterns only match the most
+# common as multiple multiline patterns are currently not possible.
+# See Also https://play.golang.org/p/ZW8CgMV-6t
```

Comment:
Each of these has "negate = true" in the example and we're using "negate = false" below. Can you update the example links with the proper configuration please?